awx/main/models/organization.py

```diff
@@ -39,6 +39,11 @@ class Meta:
         'InstanceGroup',
         blank=True,
     )
+    max_hosts = models.PositiveIntegerField(
```

Comment:
This needs documentation (`help_text`) on the field definition so it gets picked up for our OPTIONS documentation.

See: https://github.com/ansible/awx/blob/devel/awx/main/models/inventory.py#L90

Reply:
Ok, I've added something now.